pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[project]
+name = "insanely-fast-whisper"
+version = "0.0.1"
+description = "A very fast whisper CLI"
+authors = [
+    { name = "VB", email = "vaibhav@europython.eu" },
```

Comment:
should probably use another email :)

also, is it ok to add myself here?

Reply:
yes! please feel free to add yourself here! <3